webserver/webserver/src/test/java/io/helidon/webserver/BaseServerTest.java

```diff
@@ -47,16 +48,22 @@ protected static WebClient webClient() {
     }
 
     protected static void startServer(int port, Routing routing) throws Exception {
-        webServer = WebServer.builder()
+        startServer(port, routing, null);
```

Comment:
Please use `Config.empty()` instead of null - you will not need a null check and behavior will be the same

Reply:
changed